features/checkout/seeing_order_summary/seeing_addresses_on_order_summary_page.feature

```diff
@@ -0,0 +1,33 @@
+@checkout
+Feature: Seeing order addresses on order summary page
+    In order to be certain about shipping and billing address
+    As a Customer
+    I want to be able to see addresses on the order summary page
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "Lannister Coat" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And I am a logged in customer
+
+    @ui
+    Scenario: Seeing the same shipping and billing address on order summary
+        Given I have product "Lannister Coat" in the cart
+        When I specified the shipping address as "Ankh Morpork", "Frost Alley", "90210", "France" for "Jon Snow"
+        And I complete order with "Free" shipping method and "Offline" payment
+        Then I should be on the checkout summary step
+        And I should see this shipping address as shipping and billing address
```

Comment:
Agree with all.

"And this address should be used for both shipping and billing **of my order**"?

> ... store addresses per fullName in shared storage and hide the full comparison in the context. 

I would go with that approach.